src/components/MonitorLog/NonStaticQuestion.jsx

```diff
@@ -0,0 +1,293 @@
+                      <Button
+                        w="412px"
+                        h="40px"
+                        disabled={!newTitle}
+                        bgColor="ochBlue"
+                        onClick={updateQuestion}
+                      >
+                        Save Changes
+                      </Button>
+                      <Button
+                        w="412px"
+                        h="40px"
+                        bgColor="white"
+                        border="1px solid #C53030"
+                        color="#C53030"
+                        onClick={() => {
+                          editQuestionModal.onClose();
+                          deleteQuestionModal.onOpen();
+                        }}
+                      >
+                        Delete Question
+                      </Button>
+                    </VStack>
+                  </FormControl>
+                </FormControl>
+              </FormControl>
+            </Box>
+          </ModalBody>
+        </ModalContent>
+      </Modal>
+
+      {/* DELETE QUESTION MODAL STARTS HERE */}
+      <Modal isOpen={deleteQuestionModal.isOpen} onClose={deleteQuestionModal.onClose}>
+        <ModalOverlay />
+        <ModalContent h="378px">
+          <ModalHeader pl="5px" pb="0px">
+            <Button
+              bgColor="white"
+              onClick={() => {
+                deleteQuestionModal.onClose();
+                editQuestionModal.onOpen();
+              }}
+            >
+              <FiArrowLeft size="24px" />
+            </Button>
+          </ModalHeader>
+          <ModalBody>
+            <Box bgColor="rgba(253, 253, 253, 1)">
+              <Heading size="md" mb="50px">
+                Delete Question
+              </Heading>
+              <Text mt="40px">
+                Are you sure you want to delete {question.title}? This action cannot be undone.
+              </Text>
+              <Button
+                mt="70px"
+                mb="10px"
+                type="submit"
+                bgColor="ochRed"
+                color="white"
+                w="100%"
+                h="40px"
+                onClick={deleteQuestion}
+              >
+                Yes, Delete
+              </Button>
+              <Button
+                type="submit"
+                w="100%"
+                h="40px"
+                onClick={() => {
+                  deleteQuestionModal.onClose();
+                  editQuestionModal.onOpen();
+                }}
+              >
+                Cancel
+              </Button>
+            </Box>
+          </ModalBody>
+        </ModalContent>
+      </Modal>
+    </>
+  );
+}
+
+NonStaticQuestion.defaultProps = {
+  formType: null,
+  isTemplate: false,
+};
+NonStaticQuestion.propTypes = {
+  refreshTrigger: PropTypes.func.isRequired,
+  question: PropTypes.object.isRequired,
+  formType: PropTypes.string,
+  isTemplate: PropTypes.bool,
+  isDisabled: PropTypes.bool,
```

Comment:
🚫 **[eslint]** <[react/require-default-props](https://github.com/yannickcr/eslint-plugin-react/tree/master/docs/rules/require-default-props.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>propType "isDisabled" is not required, but has no corresponding defaultProps declaration.